keadm/cmd/keadm/app/cmd/edge/join.go

```diff
@@ -212,9 +213,9 @@ func join(opt *common.JoinOptions, step *common.Step) error {
 	// if edgecore start, it will get ca/certs from cloud
 	// if ca/certs generated, we can remove bootstrap file
 	err = wait.Poll(10*time.Second, 300*time.Second, func() (bool, error) {
-		if util.FileExists(edgeCoreConfig.Modules.EdgeHub.TLSCAFile) &&
-			util.FileExists(edgeCoreConfig.Modules.EdgeHub.TLSCertFile) &&
-			util.FileExists(edgeCoreConfig.Modules.EdgeHub.TLSPrivateKeyFile) {
+		if util.FileExists(path.Join(opt.CertPath, "ca/rootCA.crt")) &&
```

Comment:
```suggestion
		if util.FileExists(path.Join(opt.CertPath, "ca", "rootCA.crt")) &&
```
Use `path.Join` for all paths?